### PR TITLE
Jira-1004: New Rapid Rewards Test

### PR DESCRIPTION
**Pull request by sddgraham**

### CODE REVIEW

File: src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        if (StringUtils.containsIgnoreCase(System.getProperty(FrameworkConstants.SYSTEM_OS_NAME), "windows")) {
+            executable = PropertiesUtils.getFirstFileFromTestResources("chrome_win.exe");
+        } else if (StringUtils.containsIgnoreCase(System.getProperty(FrameworkConstants.SYSTEM_OS_NAME), "linux")) {
+            executable = PropertiesUtils.getFirstFileFromTestResources("chrome_linux_64");
+        } else {
+            executable = PropertiesUtils.getFirstFileFromTestResources("chrome_mac");
+        }
+        System.setProperty(FrameworkConstants.WEB_DRIVER_CHROME, PropertiesUtils.getPath(executable));
+    }
+
```
Comment (anastasiyakirylava) [inline src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java]: hello

File: src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    @AfterTest
+    public void afterMethod() {
+        driver.close();
+    }
+
+    @Test(retryAnalyzer = Retry.class, groups = {Groups.ACCEPTANCE_TEST})
+    public void testRapidRewardsEnrollment() {
+
+        driver.findElement(By.xpath("//button[@id='swa-header-link--rapid-rewards']")).click();
+
```
Comment (anastasiyakirylava) [inline src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java]: I HATE SPACES!!!

File: src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        suffixDropDownThree.selectByVisibleText("8");
+
+        WebElement elementFour = driver.findElement(By.xpath("//select[@id='js-customer-birth-year']"));
+        Select suffixDropDownFour = new Select(elementFour);
+        suffixDropDownFour.selectByVisibleText("1978");
+
+        WebElement elementFive = driver.findElement(By.xpath("//select[@id='js-customer-gender']"));
+        Select suffixDropDownFive = new Select(elementFive);
+        suffixDropDownFive.selectByVisibleText("Male");
+
```
Comment (anastasiyakirylava) [inline src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java]: SPACE AGAIN

File: src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java (hunk 1 above)
Comment (sddgraham) [reply src/test/java/com/airline/southwest/automation/tests/web/rewards/RapidRewardsEnrollmentTest.java]: Helllllooo

**Comment by sddgraham** [comment]
I took out 3 spaces.... are you happy now????

**Pull request merged**